yarn.lock

```diff
@@ -4877,17 +4865,7 @@ optionator@^0.8.1, optionator@^0.8.2:
     type-check "~0.3.2"
     wordwrap "~1.0.0"
 
-ora@1.3.0:
-  version "1.3.0"
-  resolved "https://registry.yarnpkg.com/ora/-/ora-1.3.0.tgz#80078dd2b92a934af66a3ad72a5b910694ede51a"
-  integrity sha1-gAeN0rkqk0r2ajrXKluRBpTt5Ro=
-  dependencies:
-    chalk "^1.1.1"
-    cli-cursor "^2.1.0"
-    cli-spinners "^1.0.0"
-    log-symbols "^1.0.2"
-
-ora@^3.2.0:
+ora@*, ora@3.4.0, ora@^3.2.0:
```

Comment:
Strange, `ora` must be a transitive dependency of something else

Reply:
I think it's from `@types/ora`. Actually I don't even think we need this module anymore since `ora` provides TypeScript definitions now - removing.

Reply:
Looks like the transitive dependency comes from `ava` though.